Makefile

```diff
@@ -68,6 +73,10 @@ test-version-bump:
 	  -installVersionOrCommit ${INSTALLTYPE} -channel ${CHANNEL} \
 	  -testCase "${TESTCASE}" -deployWorkload ${DEPLOYWORKLOAD} -workloadName ${WORKLOADNAME} -description "${DESCRIPTION}"
 
+.PHONY: test-create-mixedos
+test-create-mixedos:
+	@go test -timeout=45m -v ./entrypoint/mixedoscluster/...
+
```

Comment:
please delete all commands bellow if they are not going to be used anymore

Reply:
Will leave that to be updated as part of the Makefile changes